org.jacoco.doc/docroot/doc/repo.html

```diff
@@ -48,41 +48,71 @@
 <h2>Artifacts</h2>
 
 <p>
-  JaCoCo is split into several separate projects, the artifact ids are:   
+  Following JAR files are available:
 </p>
 
 <table class="coverage">
   <thead>
     <tr>
       <td>Group ID</td>
       <td>Artifact ID</td>
+      <td>Classifier</td>
       <td>Description</td>
     </tr>
   </thead>
   <tbody>
     <tr>
       <td><code>org.jacoco</code></td>
       <td><code>jacoco-maven-plugin</code></td>
-      <td>Plug-in for Maven builds</td>
+      <td></td>
+      <td>Plug-in for Maven</td>
     </tr>
     <tr>
       <td><code>org.jacoco</code></td>
       <td><code>org.jacoco.agent</code></td>
+      <td></td>
       <td>API to get a local copy of the agent</td>
     </tr>
     <tr>
       <td><code>org.jacoco</code></td>
+      <td><code>org.jacoco.agent</code></td>
+      <td><code>runtime</code></td>
+      <td>Agent</td>
+    </tr>
+    <tr>
+      <td><code>org.jacoco</code></td>
       <td><code>org.jacoco.ant</code></td>
+      <td></td>
       <td>Ant Tasks</td>
     </tr>
     <tr>
       <td><code>org.jacoco</code></td>
+      <td><code>org.jacoco.ant</code></td>
+      <td><code>nodeps</code></td>
+      <td>Ant Tasks <i>(all dependencies included)</i></td>
+    </tr>
+    <tr>
+      <td><code>org.jacoco</code></td>
+      <td><code>org.jacoco.cli</code></td>
+      <td></td>
+      <td>Command Line Interface</td>
+    </tr>
+    <tr>
+      <td><code>org.jacoco</code></td>
+      <td><code>org.jacoco.cli</code></td>
+      <td><code>nodeps</code></td>
```

Comment:
Re future API I meant to say that in this case we will have two versions:

1. A API version without dependencies
2. A `nodeps` version

For this situation the non-classifier version should be reserved for case 1.

I would leave documentation as it is, but repackage 3rd party dependencies in a separate PR, created #561 for this.